docs/ibm-cloud-rules.md

```diff
@@ -1230,6 +1237,71 @@ paths:
 </table>
 
 
+### Rule: delete-body
+<table>
+<tr>
+<td><b>Rule id:</b></td>
+<td><b>delete-body</b></td>
+</tr>
+<tr>
+<td valign=top><b>Description:</b></td>
+<td>This rule checks each "delete" operation and will return a warning if the operation contains a <code>requestBody</code>.</td>
+</tr>
+<tr>
+<td><b>Severity:</b></td>
+<td>warn</td>
+</tr>
+<tr>
+<td><b>OAS Versions:</b></td>
+<td>oas3</td>
+</tr>
+<tr>
+<td valign=top><b>Non-compliant example:<b></td>
+<td>
+<pre>
+paths:
+  '/v1/things/{thing_id}':
+    parameters:
+      - $ref: '#/components/parameters/ThingIdParam'
+    delete:
+      operationId: delete_thing
+      requestBody:
+          description: 'The Thing instance to be deleted.'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Thing'
+      responses:
+        '204':
+          description: 'The thing was deleted'
+</pre>
+</td>
+</tr>
+<tr>
+<td valign=top><b>Compliant example:</b></td>
+<td>
+<pre>
+paths:
+  '/v1/things/{thing_id}':
+    parameters:
+      - $ref: '#/components/parameters/ThingIdParam'
+    delete:
+      operationId: delete_thing
+      requestBody:
```

Comment:
Something tells me this request body is not here on purpose... 🙂

Reply:
I just wanted to see if you were payin' attention.  You passed :)

Reply:
fixed it